Library/Formula/elasticsearch.rb

```diff
@@ -128,6 +106,6 @@ def plist; <<-EOS.undent
   end
 
   test do
-    system "#{bin}/plugin", "--list"
+    system "#{bin}/plugin", "list"
```

Comment:
Sure thing. I've pushed such a test in commit 28d4d999a84b00878dafee56579829963592b911.